hack/examples/bases/nutanix/cluster/kustomization.yaml.tmpl

```diff
@@ -5,7 +5,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- https://raw.githubusercontent.com/nutanix-cloud-native/cluster-api-provider-nutanix/jira/KRBN-8094/templates/cluster-template-topology.yaml
+- https://raw.githubusercontent.com/nutanix-cloud-native/cluster-api-provider-nutanix/main/templates/cluster-template-topology.yaml
```

Comment:
This is reverting from `main` maybe you can drop this commit and rebase instead?